components/suggestion/command_provider/command_provider.tsx

```diff
@@ -194,6 +194,7 @@ export default class CommandProvider extends Provider {
                             Hint: hint,
                             Description: cmd.auto_complete_desc,
                             IconData: '',
+                            type: Constants.Integrations.COMMAND,
```

Comment:
I was going to say that we also need this change on line 173 and 240, for slash command suggestions from the App framework, but that is handled in `app_command_parser.ts` now. A bit confusing when someone is reading only this file, but the logic sound :+1: